ci-operator/config/openshift-eng/ocp-qe-perfscale-ci/openshift-eng-ocp-qe-perfscale-ci-main__metal-4.18-latest-dataplane-x86.yaml

```diff
@@ -0,0 +1,84 @@
+build_root:
+  image_stream_tag:
+    name: ci-tools-build-root
+    namespace: ci
+    tag: latest
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.18"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- always_run: false
+  as: regulus-cpt
+  capabilities:
+  - intranet
+  restrict_network_access: false
+  steps:
+    cluster_profile: metal-perscale-cpt
+    env:
+      BAREMETAL: "true"
+    test:
+    - ref: openshift-qe-installer-bm-load-kubeconfig
+    - ref: openshift-qe-cluster-health
+    - ref: openshift-qe-installer-bm-day2-regulus
+- always_run: false
+  as: regulus-jetlag
+  capabilities:
+  - intranet
+  restrict_network_access: false
+  steps:
+    cluster_profile: metal-perfscale-jetlag
+    env:
+      BAREMETAL: "true"
+    test:
+    - ref: openshift-qe-installer-bm-load-kubeconfig
+    - ref: openshift-qe-cluster-health
+    - ref: openshift-qe-installer-bm-day2-regulus
+- always_run: false
+  as: regulus-long-cpt
+  capabilities:
+  - intranet
+  restrict_network_access: false
+  steps:
+    cluster_profile: metal-perscale-cpt
+    env:
+      BAREMETAL: "true"
+      CRUCIBLE: "true"
+      JETLAG_BRANCH: main
+      NUM_WORKER_NODES: "2"
```

Comment:
Remove NUM_WORKER_NODES var, is not needed anymore, see: https://github.com/openshift/release/pull/66657